browser.js

```diff
@@ -310,6 +310,11 @@ document.addEventListener('DOMContentLoaded', () => {
 	// enable OS specific styles
 	document.documentElement.classList.add(`os-${process.platform}`);
 
-	// TODO: figure out a better way to detect when React is done
-	setTimeout(init, 300);
+	// detect when React is ready before firing init
+	const awaitReact = setInterval(() => {
```

Comment:
This is using `querySelector` so it's only getting the first matching element, and we (currently) have no reason to ask for more. I would only say that we're looking for _at least one_ of some item ever anyway. I can't think of any scenario where you'd need a whole list of nodes. For example, you won't click more than 1 thing at a time. I could change it to a `querySelectorAll`, but I don't see why we need that. Feel free to point out something I'm missing. 😄